images/update-dbg/Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM golang:1.18 AS pullrequestcreator
+FROM golang:1.20 AS gobuilder
```

Comment:
`pr-creator` now needs at least golang:1.19; given that golang:1.20 has glibc version that is too new to be used in debian:buster, compile with CGO disabled.
We will need to port this fix in all other images using `pr-creator` from test-infra.